qiskit_experiments/calibration/backend_calibrations.py

```diff
@@ -0,0 +1,141 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Store and manage the results of a calibration experiments in the context of a backend."""
+
+from datetime import datetime
+from typing import Any, Dict, List, Union
+import copy
+
+from qiskit.providers.ibmq.ibmqbackend import IBMQBackend as Backend
+from qiskit.providers import BaseBackend
+from qiskit.circuit import Parameter
+from qiskit_experiments.calibration.calibrations import Calibrations, ParameterKey
+
+
+class BackendCalibrations(Calibrations):
+    """
+    A Calibrations class to enable a seamless interplay with backend objects.
+    This class enables users to export their calibrations into a backend object.
+    Additionally, it creates frequency parameters for qubits and readout resonators.
+    The parameters are named `qubit_freq_est` and `meas_freq_est` to be consistent
+    with the naming in backend.defaults(). These two parameters are not attached to
+    any schedule.
+    """
+
+    def __init__(self, backend: Union[Backend, BaseBackend]):
+        """Setup an instance to manage the calibrations of a backend."""
+        super().__init__(backend.configuration()._control_channels)
```

Comment:
I do not know why. Would it be worth it changing this in terra for this particular use case?

Reply:
That seems reasonable. It looks like it is only used to support [the `control` method](https://github.com/Qiskit/qiskit-terra/blob/98130dd6158d1f1474e44dd5aeacbc619174ad63/qiskit/providers/models/backendconfiguration.py#L737) that lets you query a single set of qubits at a time. You only ever query a single set of qubits at a time in this PR but it is nice for `Calibrations` to require only a `dict` instead of a `BackendConfiguration` instance.

Reply:
Opened an issue here: https://github.com/Qiskit/qiskit-terra/issues/6326